test_app_database.py

```
@@ -0,0 +1,244 @@
+    assert existing_conversation is not None
+
+    existing_conversation.last_used_model = model
+    existing_conversation.last_used_provider = provider
+    existing_conversation.last_message_at = datetime.now(UTC)
+    existing_conversation.message_count += 1
+    existing_conversation.topic_summary = topic_summary
+    session.commit()
+
+
+def test_store_new_user_conversations_small_db(
+    sqlite_database: Callable, benchmark: BenchmarkFixture
```

Ruff (0.14.14)

[warning] Unused function argument: `sqlite_database`

(ARG001)